tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "sourceMap": true,
+    "target": "es2018",
+    "module": "esnext",
+    "lib": ["esnext"],
+    "esModuleInterop": true,
+    "moduleResolution": "node",
+    "outDir": "lib",
+    "declaration": false,
```

Comment:
Why?

Reply:
siroc uses rollup plugin dts to produce bundled declaration files with some key advantages:
* smaller size
* no 'phantom' types for purely internal exports